### PR TITLE
Selenium: Rework the 'RunPlainJavaProjectTest' to use the 'kohsuke Github' library

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Rework the 'RunPlainJavaProjectTest' to use the 'kohsuke Github' library
* Add the 'run-plain-java-project' to the resources
* Delete the 'KeepDirectotryGitImportTest' from the 'SmokeTestsSuite' as non already exist
* Delete the 'PushChangeNotUpdatedRepoTest' from the project as a duplicate of the 'PushingChangesTest'

### What issues does this PR fix or reference?
#9458

### CODE REVIEW

**Comment by artaleks9** [comment]
ci-test

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/plainjava/RunPlainJavaProjectTest.java [hunk 1]
```diff
@@ -95,55 +110,39 @@ public void checkRunPlainJavaProject() {
...
+    // check code assist when use library
     projectExplorer.openItemByPath(PROJECT_NAME + "/src/com/company/nba/MainClass.java");
     codenvyEditor.waitActive();
-    codenvyEditor.setCursorToLine(9);
+    codenvyEditor.setCursorToLine(19);
     codenvyEditor.typeTextIntoEditor(Keys.TAB.toString());
     codenvyEditor.typeTextIntoEditor("Mockito mockito = new Mockito();");
     codenvyEditor.waitTextIntoEditor("Mockito mockito = new Mockito();");
-    codenvyEditor.waitMarkerInPosition(ERROR, 9);
+    codenvyEditor.waitMarkerInPosition(ERROR, 19);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/plainjava/RunPlainJavaProjectTest.java:120]: Please, divide separate steps by empty lines.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/plainjava/RunPlainJavaProjectTest.java [hunk 2]
```diff
@@ -95,55 +110,39 @@ public void checkRunPlainJavaProject() {
...
-    askForValueDialog.clickOkBtnNewJavaClass();
-    askForValueDialog.waitNewJavaClassClose();
-    projectExplorer.waitItem(PROJECT_NAME + "/src/base/test/A.java");
+    createNewPackageAndJavaClass();
+
+    // check the code assist when create the instance of new Java class
     projectExplorer.openItemByPath(PROJECT_NAME + "/src/com/company/nba/MainClass.java");
     codenvyEditor.waitActive();
-    codenvyEditor.setCursorToLine(12);
+    codenvyEditor.setCursorToLine(22);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/plainjava/RunPlainJavaProjectTest.java]: Please, divide separate steps by empty lines.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/plainjava/RunPlainJavaProjectTest.java [hunk 3]
```diff
@@ -218,9 +212,24 @@ private void importPlainJavaApp(String url, String nameApp, String typeProject)
...
     projectWizard.waitExpTextInSourceFolder("store", Wizard.TypeFolder.LIBRARY_FOLDER);
     projectWizard.clickSaveButton();
-    loader.waitOnClosed();
     projectWizard.waitCloseProjectConfigForm();
     projectExplorer.waitVisibilityByName(nameApp);
-    loader.waitOnClosed();
+  }
+
+  private void createNewPackageAndJavaClass() {
+    projectExplorer.waitAndSelectItem(PROJECT_NAME + "/src");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/plainjava/RunPlainJavaProjectTest.java:224]: Please, divide separate steps by empty lines.

**Review by SkorikSergey — APPROVED** [review]

**Review by dmytro-ndp — APPROVED** [review]

**Comment by artaleks9** [comment]
ci-test

**Comment by artaleks9** [comment]
ci-test

**Pull request merged**